data-prepper-plugins/saas-source-plugins/crowdstrike-source/src/main/java/org/opensearch/dataprepper/plugins/source/crowdstrike/models/CrowdStrikeIndicatorResult.java

```diff
@@ -3,16 +3,20 @@
 import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Getter;
+import lombok.Setter;
+
 import java.util.List;
 
 /**
  * The result of Falcon query search.
  */
 @Getter
+@Setter
```

Comment:
Minor: `@Data` contains both `@Getter` and `@Setter` functionality + has the added bonus of providing a `toString` implementation for logging